projects/roscoe/roscoe_data/download_annotated.sh

```diff
@@ -1,4 +1,38 @@
 #!/bin/bash
-# Human annotated data: Pending data release approval
-# this file will contain a command to download annotated datasets into "roscoe_data/annotated" folder
-echo "Sorry, Pending data release approval"
+
+PATH_TO_DATA="./projects/roscoe/roscoe_data"
```

Comment:
nit; Usually data is downloaded to `ParlAI/data/{project}`, where the path to that `data/` folder is found in `opt['datapath']`. I think one reason is that `data/` is in the .gitignore (and there may be other assumptions coded in, in other places).